content/en/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Kubernetes API health endpoints
+reviewers:
+- logicalhan
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+This page describes the usage of the Kubernetes API endpoints for liveness and readiness.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## API Endpoints for health
+
+Currently the Kubernetes API server supports 3 API endpoints (`healthz`, `livez`, `readyz`) to indicate the current status of the API server.
+Since Kubernetes 1.16 the `healthz` endpoint is deprecated and the more specific `livez` and `readyz` should be used.
```

Comment:
Phrasing nit (feel free to merge as-is):
```suggestion
The `healthz` endpoint is deprecated (since Kubernetes v1.16), and you should use the more specific `livez` and `readyz` endpoints instead.
```

(_How_ do people use these endpoints? For example: do they invoke `k8s.makeInformer()` with that path? The page as drafted is not really clear what kind of response you can negotiate or whether the body of the response is supposed to be machine-readable).